app/models/turbo/streams/tag_builder.rb

```diff
@@ -32,108 +32,127 @@ def initialize(view_context)
 
   # Removes the <tt>target</tt> from the dom. The target can either be a dom id string or an object that responds to
   # <tt>to_key</tt>, which is then called and passed through <tt>ActionView::RecordIdentifier.dom_id</tt> (all Active Records
-  # do). Examples:
+  # do), or a CSS selector string (when using <tt>target_multiple: true</tt>). Examples:
   #
   #   <%= turbo_stream.remove "clearance_5" %>
   #   <%= turbo_stream.remove clearance %>
-  def remove(target)
-    action :remove, target, allow_inferred_rendering: false
+  #   <%= turbo_stream.remove ".invalid-row", target_multiple: true %>
+  def remove(target, target_multiple: false)
```

Comment:
I'd rather see us add a separate `#remove_all(targets)` method than overloading this one.